### PR TITLE
RbzTable and Dashboard style changes

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/components/QuarantinedList.vue [hunk 1]
```diff
@@ -150,11 +150,11 @@ export default defineComponent({
           fieldNames: ['tags'],
           isSortable: true,
           displayFunction: (item: Quarantined) => {
-            return item.tags?.join('\n')
+            return item.tags?.length
           },
           isSearchable: true,
           classes: 'width-110px',
-          cellContentClasses: 'vertical-scroll ellipsis',
+          cellContentClasses: 'multi-line ellipsis',
```
Comment (Aviv-Galmidi) [inline src/components/QuarantinedList.vue:157]: We don't need `multi-line` if it's just a static number (length) that doesn't have multiple lines

File: src/components/RbzTable.vue [hunk 2]
```diff
@@ -642,4 +644,15 @@ export default defineComponent({
 .rbz-table .data-cell-content {
   max-height: 1.75rem;
 }
+
+.pagination-buttons {
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:648]: `pagination-button` is preferred

File: src/components/RbzTable.vue [hunk 3]
```diff
@@ -642,4 +644,15 @@ export default defineComponent({
...
+  background: none !important;
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:649]: Avois using `!important` whenever possible as it's not a good practice to use it, if needed find a stronger selector

File: src/components/RbzTable.vue [hunk 4]
```diff
@@ -642,4 +644,15 @@ export default defineComponent({
...
+  border: 0;
+  padding: 0 !important;
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:651]: Avois using `!important` whenever possible as it's not a good practice to use it, if needed find a stronger selector

File: src/components/RbzTable.vue [hunk 5]
```diff
@@ -642,4 +644,15 @@ export default defineComponent({
...
+}
+
+.is-button-active {
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:654]: `pagination-button-active` would be better, as it's a specific styling for a pagination button when it's active and not any button

File: src/components/RbzTable.vue [hunk 6]
```diff
@@ -642,4 +644,15 @@ export default defineComponent({
...
+  color: #069;
```
Comment (Aviv-Galmidi) [inline src/components/RbzTable.vue:655]: Let's move this color to `colors.scss`

File: src/views/Dashboards.vue [hunk 7]
```diff
@@ -171,8 +171,683 @@ export default defineComponent({
             'Content-Type': 'application/json',
           },
         },
-      })
-      this.data = response?.data?.data?.results || []
+      }) */
+      // TODO: this.data = response?.data?.data?.results || []
```
Comment (Aviv-Galmidi) [inline src/views/Dashboards.vue:175]: Don't forget to remove this `TODO` when you're done

File: src/views/DocumentList.vue [hunk 8]
```diff
@@ -166,6 +166,7 @@ export default defineComponent({
       selectedDocMatchingGlobalFilter: null as GlobalFilter,
       matchedDocsData: null as any,
       customResponsesNames: [] as [CustomResponse['id'], CustomResponse['name']][],
+      counter: 0,
```
Comment (Aviv-Galmidi) [inline src/views/DocumentList.vue:169]: What's this? 
Added by accident?

File: src/components/QuarantinedList.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/components/QuarantinedList.vue:157]: Done

File: src/components/RbzTable.vue (hunk 2 above)
Comment (AvihayMReblaze) [reply src/components/RbzTable.vue:648]: Done

File: src/components/RbzTable.vue (hunk 3 above)
Comment (AvihayMReblaze) [reply src/components/RbzTable.vue:649]: Done

File: src/components/RbzTable.vue (hunk 4 above)
Comment (AvihayMReblaze) [reply src/components/RbzTable.vue:651]: Done

File: src/views/Dashboards.vue (hunk 7 above)
Comment (AvihayMReblaze) [reply src/views/Dashboards.vue:175]: Done

File: src/views/DocumentList.vue (hunk 8 above)
Comment (AvihayMReblaze) [reply src/views/DocumentList.vue:169]: Accident, done.

File: src/components/RbzTable.vue (hunk 6 above)
Comment (AvihayMReblaze) [reply src/components/RbzTable.vue:655]: Done

**Review by Aviv-Galmidi — DISMISSED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**